mod_reforged/hooks/config/item_names.nut

```diff
@@ -1,5 +1,12 @@
 // Vanilla Adjustments
+::MSU.Array.removeValues(::Const.Strings.BillNames, [
+	"Reach",	// This is re-used multiple times. We restrict it to just Bows
```

Comment:
Bardiche and Voulge removal are fine. No need to remove Reach from any of the names.